src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
```

Comment:
@AmosWels  Thanks for the good work, however, I am suggesting we do away with the code from line 9 to line 22 so that our app looks fresh. You could also add an h1 tag with "welcome to authors haven" to replace the code from line 9 to line 22.

Reply:
@zellymiriam, Thanks for the comment, will look into modifying it. However much of this work will be replaced with the landing page of the application from the UI that @zaabu is currently working on.

Reply:
alright